File: frontend/src/app/admin/dto/config-item-response.ts
export type ConfigDataType = 'STRING' | 'INTEGER' | 'DECIMAL' | 'BOOLEAN';

export interface ConfigItemResponse {
  id: number;
  configKey: string;
  configValue: string;
  dataType: ConfigDataType;
  description?: string;
  configGroup?: string;
}

File: backend/src/main/java/com/ngv/libraryManagementSystem/dto/response/ConfigItemResponse.java
package com.ngv.libraryManagementSystem.dto.response;

import com.ngv.libraryManagementSystem.enums.ConfigDataTypeEnum;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ConfigItemResponse {
    private Long id;
    private String configKey;
    private String configValue;
    private ConfigDataTypeEnum dataType;
    private String description;
    private String configGroup;
}

